packages/react-components/react-spinbutton/src/stories/SpinButton/SpinButtonAccessibility.md

```diff
@@ -49,10 +49,18 @@ When a user holds down an edit key or mouse click, there is an animation of spin
 
 ## Known issues
 
-Placeholder: If there are any current known bugs with browsers or AT that impact this component, list them here.
-
 Mouse press quick changes to value are not announced to screen readers.
 
+[NVDA does not read the value of spinbuttons in Chromium](https://github.com/nvaccess/nvda/issues/13195)
+
+[Narrator defaults to reading min/max as 0 when they're (intentionally) undefined](https://microsoft.visualstudio.com/Edge/_workitems/edit/39070743)
+
+### Narrator + Edge
+When no min or max values are set Narrator announces "minimum 0" and "maximum 0". This is misleading because when these values are not set SpinButton does not enforce a min or max value. Example.
```

Comment:
```suggestion
When no min or max values are set Narrator announces "minimum 0" and "maximum 0". This is misleading because when these values are not set SpinButton does not enforce a min or max value.
```